### PR TITLE
[zh-cn] update cncf-code-of-conduct

### PR DESCRIPTION
**Pull request by yanrongshi**

### CODE REVIEW

**Comment by tengqm** [comment]
请用注释将英文原文补上。

**Comment by yanrongshi** [comment]
> 请用注释将英文翻译成补上。

好的，已更新，感谢提醒

**Review by sftim — COMMENTED** [review]
**Do not edit this file directly. Get the latest from https://github.com/cncf/foundation/blob/master/code-of-conduct-languages/zh.md**

/hold

This change needs to go upstream first.

**Comment by yanrongshi** [comment]
> **不要直接编辑此文件。[从https://github.com/cncf/foundation/blob/master/code-of-conduct-languages/zh.md](https://github.com/cncf/foundation/blob/master/code-of-conduct-languages/zh.md)获取最新信息[](https://github.com/cncf/foundation/blob/master/code-of-conduct-languages/zh.md)**
> 
> /抓住
> 
> 这种变化需要先到上游。

Okay, next, I should close this PR, right？

File: content/zh-cn/community/static/cncf-code-of-conduct.md [hunk 1]
```diff
@@ -1,39 +1,112 @@
 <!-- Do not edit this file directly. Get the latest from
-     https://github.com/cncf/foundation/blob/master/code-of-conduct-languages/zh.md -->
-## 云原生计算基金会（CNCF）社区行为准则 1.0 版本
+     https://github.com/cncf/foundation/blob/main/code-of-conduct-languages/zh.md -->
+<!-- ## CNCF Community Code of Conduct v1.1 -->
+## 云原生计算基金会（CNCF）社区行为准则 1.1 版本
```
Comment (Sea-n) [inline content/zh-cn/community/static/cncf-code-of-conduct.md:4]: Please take a look at the first line.

**Review by Sea-n — COMMENTED** [review]
This file should not be translated here. It's just copy of cncf foundation one.

Hold until the changes is merged in upstream repo.
/hold

File: content/zh-cn/community/static/cncf-code-of-conduct.md (hunk 1 above)
Comment (yanrongshi) [reply content/zh-cn/community/static/cncf-code-of-conduct.md:4]: nice,thanks

**Pull request closed without merging**